# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/ash/src/extensions/khr/mod.rs
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/third_party/rust/ash/src/extensions/khr/mod.rs b/third_party/rust/ash/src/extensions/khr/mod.rs
--- a/third_party/rust/ash/src/extensions/khr/mod.rs
+++ b/third_party/rust/ash/src/extensions/khr/mod.rs
@@ -1,37 +1,53 @@
 pub use self::acceleration_structure::AccelerationStructure;
 pub use self::android_surface::AndroidSurface;
+pub use self::buffer_device_address::BufferDeviceAddress;
 pub use self::create_render_pass2::CreateRenderPass2;
 pub use self::deferred_host_operations::DeferredHostOperations;
 pub use self::display::Display;
 pub use self::display_swapchain::DisplaySwapchain;
 pub use self::draw_indirect_count::DrawIndirectCount;
+pub use self::external_fence_fd::ExternalFenceFd;
 pub use self::external_memory_fd::ExternalMemoryFd;
+pub use self::external_semaphore_fd::ExternalSemaphoreFd;
+pub use self::get_memory_requirements2::GetMemoryRequirements2;
+pub use self::get_physical_device_properties2::GetPhysicalDeviceProperties2;
+pub use self::maintenance1::Maintenance1;
+pub use self::maintenance3::Maintenance3;
 pub use self::pipeline_executable_properties::PipelineExecutableProperties;
 pub use self::push_descriptor::PushDescriptor;
 pub use self::ray_tracing_pipeline::RayTracingPipeline;
 pub use self::surface::Surface;
 pub use self::swapchain::Swapchain;
+pub use self::synchronization2::Synchronization2;
 pub use self::timeline_semaphore::TimelineSemaphore;
 pub use self::wayland_surface::WaylandSurface;
 pub use self::win32_surface::Win32Surface;
 pub use self::xcb_surface::XcbSurface;
 pub use self::xlib_surface::XlibSurface;
 
 mod acceleration_structure;
 mod android_surface;
+mod buffer_device_address;
 mod create_render_pass2;
 mod deferred_host_operations;
 mod display;
 mod display_swapchain;
 mod draw_indirect_count;
+mod external_fence_fd;
 mod external_memory_fd;
+mod external_semaphore_fd;
+mod get_memory_requirements2;
+mod get_physical_device_properties2;
+mod maintenance1;
+mod maintenance3;
 mod pipeline_executable_properties;
 mod push_descriptor;
 mod ray_tracing_pipeline;
 mod surface;
 mod swapchain;
+mod synchronization2;
 mod timeline_semaphore;
 mod wayland_surface;
 mod win32_surface;
 mod xcb_surface;
 mod xlib_surface;